src/index.js

```diff
@@ -69,55 +69,37 @@ var rangeColor = {
   component: 'color-picker',
   dualOutput: true
 };
-var middleThreshRangeColor = {
-  ref: 'props.section4.middleThreshRangeColor',
-  label: 'Change middle range gradient',
-  type: 'integer',
-  defaultValue: 0.7,
-  min: 0,
-  max: 1.55,
-  step: .05,
-  component: 'slider'
-};
-var lowerThreshRangeColor = {
-  ref: 'props.section4.lowerThreshRangeColor',
-  label: 'Change lower range gradient',
-  type: 'integer',
-  defaultValue: 0.85,
-  min: 0,
-  max: 1.55,
-  step: .05,
-  component: 'slider'
-};
 var lowerThreshRange = {
   ref: 'props.section4.lowerThreshRange',
-  label: 'Change lower range',
-  type: 'integer',
-  defaultValue: 0.5,
+  label: 'Set lower range (%)',
+  type: 'number',
+  defaultValue: 50,
   min: 0,
-  max: 1.05,
-  step: .05,
-  component: 'slider'
+  max: 100,
+  change: function(data){
```

Comment:
({ props: { section4 }) => {

could go another level grabbing the actual props, but section4 might add some value to see there